packages/tmp-foundry-sdk-generator/src/ontologyMetadata/ontologyMetadataResolver.ts

```diff
@@ -153,27 +208,55 @@ export class OntologyMetadataResolver {
       ]);
     }
 
+    const externalObjects = new Map();
+    const externalInterfaces = new Map();
+
+    for (const { sdk } of extPackageInfo.values()) {
+      if (sdk.npm?.npmPackageName == null) {
+        throw new Error("External package is not generated as an npm package");
+      }
+
+      const dataScope = sdk.inputs.dataScope.ontologyV2;
+
+      for (const rid of dataScope.objectTypes) {
+        const ot = Object.values(ontologyFullMetadata.objectTypes).find(
+          (ot) => ot.objectType.rid === rid,
+        );
+
+        if (!ot) {
+          throw new Error(
+            `Could not find external object type with rid ${rid}`,
```

Comment:
nit: Maybe add to this error the ontology rid that we're looking in, could imagine some people getting confused by this error if for some reason they tried including an object type for another ontology. I'm just spitballing here tho